css-three-column-layout/styles.css

```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+body {
+  color: white;
+  font-family: sans-serif, 'Courier New', Courier;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  background-color: rgb(25, 150, 218);
+}
+
+.row {
+  display: flex;
+  padding: 0 70px;
```

Comment:
You should make this `padding` a separate class from `row`. The only styling applied to a `row` class should be `display: flex`